pages/common/ipfs.md

```diff
@@ -0,0 +1,44 @@
+# ipfs
+
+> Inter Planetary File System.
+> A peer-to-peer hypermedia protocol to make the web faster, safer, and more open.
+
+- Save a remote file and give It a name but not pin It:
+
+`ipfs get {{hash}} -o {{filename}}`
```

Comment:
It is actually the hash of the file... the IPFS is all about these... A file is retrieved inserting the hash and not the address to the file.

Reply:
So we should mention the add examples first. Otherwise, the user does not know from where to get the file hash. We always arrange the commands in the order of usage, if a command needs the output of another command we need to mention that first.

Reply:
Perfect, going to modify with this style =)